### PR TITLE
Sign CLA

### PR DESCRIPTION
**Pull request by prayagverma**

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -2,9 +2,10 @@ By adding your name and email address you agree to the terms of Lyft's CLA.
...
-1. Aleksei Zakharov (alexey.v.zaharov@gmail.com)
-1. Ostap Taran (austinate@me.com)
-1. Zachary Perrault (zach.perrault@gmail.com)
+2. Evelio Tarazona (evelio@evelio.info)
+3. Bart Whiteley (bwhiteley@gmail.com)
+4. skyfishjy (skyfish.jy@gmail.com)
+5. Aleksei Zakharov (alexey.v.zaharov@gmail.com)
+6. Ostap Taran (austinate@me.com)
+7. Zachary Perrault (zach.perrault@gmail.com)
+8. Prayag Verma (prayag.verma@gmail.com)
```
Comment (keith) [inline CONTRIBUTORS.md]: You can make all of these 1. Markdown renders them correctly regardless of the number

**Comment by keith** [comment]
Will merge after numbers change, thanks!

**Comment by prayagverma** [comment]
@keith Updated the pull request to make all of them as 1

**Comment by keith** [comment]
Thanks!

**Pull request merged**